File: c++/Disk.cpp
class Disk{
	private:
		/*atribut private*/
		string type;
		string capacity;
		string price;

	public:
		/*constructor*/
		Disk(){}
		Disk(string type, string capacity, string price){
			this->type = type;
			this->capacity = capacity;
			this->price = price;
		}

		/*getter/setter*/
		void settype(string type){
			this->type = type;
		}
		string gettype(){
			return this->type;
		}
		void setcapacity(string capacity){
			this->capacity = capacity;
		}
		string getcapacity(){
			return this->capacity;
		}
		void setprice(string price){
			this->price = price;
		}
		string getprice(){
			return this->price;
		}

		/*Method*/
		void printDisk(){
			cout << "Disk type : " << this->type << endl;
			cout << "Disk capacity : " << this->capacity << endl;
			cout << "Disk price : " << this->price << endl;
		}
		/*destructor*/
		~Disk(){}
};

File: php/Disk.php
<?php

class Disk{
	// atribut private
    private $type;
    private $capacity;
    private $price;

	// constructor
	public function __construct($type, $capacity, $price){
        $this->type = $type;
        $this->capacity = $capacity;
        $this->price = $price;
	}

	// setter/getter
    public function settype($type){
		$this->type = $type;
	}
    public function gettype(){
		return $this->type;
	}

    public function setcapacity($capacity){
		$this->capacity = $capacity;
	}
    public function getcapacity(){
		return $this->capacity;
	}

    public function setprice($price){
		$this->price = $price;
	}
    public function getprice(){
		return $this->price;
	}

	/*Method*/
	public function printDisk(){
		echo "Disk type : ".$this->type."<br>";
		echo "Disk capacity : ".$this->capacity."<br>";
		echo "Disk price : ".$this->price."<br>";
	}
}

?>